source/_posts/2024-06-11-roadmap-2024h1.markdown

```diff
@@ -0,0 +1,162 @@
+
+#### Current priority 1: Make automations easier to organize
+
+As users grow their smart home, they will create more automation. Our automations are listed in a large table, and it can get easily disorganized as it grows. Earlier this year we did a lot to improve the organization of automations, including [labels, filters, and categories](https://youtu.be/g3Xo0ArcMaY?si=OrqvNDASLFvyKhEs), which in return not only made automations easier to organize, but also almost every table in the Home Assistant settings.
+
+#### Current priority 2: Make automations easier to create
+
+While we have been making headway earlier this year making automations easier to create. We need a more comprehensive effort with a full product development cycle of research, discovery, ideation, and validation to confidently make design and implementation decisions. As of now, many seemingly simple automations can be too complex to create and debug, and there is a lot of room for improvement. We foresee that when this opportunity is done, admins and housemates will be able to create automations with ease.
+
+* * *
+
+### Research
+
+To make informed product decisions, we need to base our work on solid research. Therefore, we have been building out our research capabilities, both in terms of user research and technology research.
+
+#### User research
+
+In order for all of us in our community to make the right product, design, and engineering decisions, we need all of us to have a clear picture of who our users really are. Last year, we had done some initial research, mostly focused on dashboard. This year, we want to get the full picture with complete foundational research of our users.
+
+Secondly, of the three Open Home values, privacy, choice, and sustainability, while we have a pretty solid understanding of choice and some understanding of sustainability, the idea of privacy can benefit from more clarity. We want a better understanding of what privacy really means to a home and its residents.
+
+#### Technology research (R&D)
+
+Yes, large language models (LLMs) are a hot trend, but that’s not why we’re exploring them. We know there are very interesting problems they can solve in the home, and with the amount of data our users collect in their history, we believe that there is much power underneath Home Assistant yet to be unearthed and it is uniquely positioned to be the smart home platform for AI. [Our AI vision](/blog/2024/06/07/ai-agents-for-the-smart-home/) lays down the foundation for what we want to do with AI and LLMs next.
+
+In order to help AI make better decisions, we need a better understanding of the context of what each device actually is. To do so, we need to refine our information architecture. For instance, we recently rolled out floors helping Home Assistant better understand your house. This is no small task but could be vital in implementing better voice or even AI in the home.
```

Comment:
```suggestion
In order to help AI make better decisions, we need a better understanding of the context of what each device actually is. To do so, we need to refine our information architecture. For instance, we recently rolled out floors helping Home Assistant better understand your house. This is no small task, but it could be vital in implementing better voice or even AI in the home.
```